### DISCUSSION TITLE
Starting the Tool after installation

### DISCUSSION THREAD

**Discussion by TimonPetermann** [ThalesGroup/xsmp-modeler-core#12] [Q&A] [answered]
Hi,

how can I run the tool after`mvn install` succeded?

Thank you for you help!

Cheers, Timon

**Comment by ydaveluy** [comment, marked as answer]
Hi Timon,
The mvn install should generate a zip file in org.eclipse.xsmp.repository/target/org.eclipse.xsmp.repository-xxx.zip
From a running eclipse you can run Help --> Install new software ... --> Add --> Archive and select the previous zip file. Then you can install XSMP.
XSMP is still in development, an official release should be available in march I hope :-)

Cheers,

Yannick

**Comment by ydaveluy** [comment]
The [wiki](https://github.com/ThalesGroup/xsmp-modeler-core/wiki/User-guide#installation) may help.

**Comment by TimonPetermann** [comment]
Thanks a lot!

**Comment by ydaveluy** [comment]
You are probably one of the first external user. 

Any feedback are welcome.
